src/qt/addresstablemodel.cpp

```diff
@@ -228,6 +228,16 @@ QVariant AddressTableModel::data(const QModelIndex &index, int role) const
         default: break;
         }
     }
+    else if (role == Qt::ForegroundRole) {
+        if (rec->type == AddressTableEntry::Receiving && walletModel->wallet().isUsedAddress(DecodeDestination(rec->address.toStdString()))) {
+            return QColor(Qt::lightGray);
```

Comment:
Suggestion: perhaps `gray` to be more compatible with dark mode.

gray:
![Screenshot from 2020-06-07 10-47-18](https://user-images.githubusercontent.com/2415484/83964322-d1b63d00-a89b-11ea-8294-f365f60395ae.jpg)

light gray:
![Screenshot from 2020-06-07 10-41-39](https://user-images.githubusercontent.com/2415484/83964324-d4b12d80-a89b-11ea-86a7-39b2845ac950.jpg)

Reply:
Thanks for testing with dark mode! have now updated the code to use `gray` instead of `lightGray` to accommodate both light and dark modes